src/_variables.scss

```diff
@@ -1,12 +1,33 @@
-$lightest-blue: #e7f0ff;
-$lighter-blue: #d3e3fc;
-$light-blue: #b1cefd;
-$blue: #77a6f7;
+$blue: (
+  blue-baby: #00a5bb,
+  light: #b1cefd,
+  lighter: #d3e3fc,
+  lightest: #e7f0ff,
+  normal: #77a6f7
+);
+
+$gray: (
+  200: #e9ecef,
+  300: #dee2e6,
+  800: #343a40,
+  dark: #333,
```

Comment:
Maybe these should be renamed to lightest, lighter etc. to be more clear? Also, `800` and `dark` are very similar, they could be merged into one.